Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -2883,6 +2883,42 @@ static void SetMetadataParameterValues_DEPRECATED(
 PRAGMA_ENABLE_DEPRECATION_WARNINGS
 #pragma endregion
 
+namespace {
+void addInstanceFeatureIds(UCesiumGltfInstancedComponent* pInstancedComponent) {
+  const TSharedPtr<FCesiumPrimitiveFeatures>& pInstanceFeatures =
+      pInstancedComponent->pInstanceFeatures;
+  if (!pInstanceFeatures) {
+    return;
+  }
+  const TArray<FCesiumFeatureIdSet>& featureIdSets =
+      UCesiumPrimitiveFeaturesBlueprintLibrary::GetFeatureIDSets(
+          *pInstanceFeatures);
+  int32 featureSetCount = featureIdSets.Num();
+  if (featureSetCount == 0) {
+    return;
+  }
+#if ENGINE_VERSION_5_3_OR_HIGHER
+  pInstancedComponent->SetNumCustomDataFloats(featureSetCount);
+#else
+  pInstancedComponent->NumCustomDataFloats = featureSetCount;
+#endif
```

Comment:
Looks like the `M_CesiumBaseMaterial` has been saved with UE 5.3 in this branch. That's fine, because our next release will be 5.3+ only. But given that that is the case, this `#if` is unnecessary.

```suggestion
  pInstancedComponent->SetNumCustomDataFloats(featureSetCount);
```